app/views/records/edit_fields/_rights.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= f.input :rights, as: :select_with_modal_help,
+<%= f.input :rights, as: :select,
     collection: RightsService.select_options,
-    input_html: { class: 'form-control', multiple: true } %>
+    input_html: { class: 'form-control' } %>
```

Comment:
@njaffer Has the rights changed from a multi select value to a single value? There is no indication in sufia 6.0 that this has changed: https://github.com/projecthydra/sufia/blob/6.x-stable/sufia-models/app/models/concerns/sufia/generic_file/metadata.rb#L41 

@mjgiarlo was this a decision that was made to move from multiple rights statements to a singular value?